Week2/todo_app/program.js

```diff
@@ -16,43 +16,139 @@ class TodoModel {
                 callback(err);
                 return;
             }
-
             callback(null, results);
         });
     }
 
-    create(description, callback) {
+    create(description, user_id, callback) {
         // Write code and query to create a new TODO item
+        const createQuery = `
+        INSERT INTO todo_items(text, is_completed, user_id) VALUES("${description}" , 0 , ${user_id})
```

Comment:
This is still open to SQL injection, so you could use escaping or prepared statements to protect from that.

Reply:
I thing I was doing that but i will check it out again.